app/models.py

```diff
@@ -86,6 +87,7 @@ def serialize(self):
             'slug': self.slug,
             'framework': self.framework,
             'status': self.status,
+            'clarification_questions_open': self.clarification_questions_open,
```

Comment:
Good call. I've renamed both, but keeping `one_service_limit` too for now since it's used by the supplier frontend.